src-ext/TypeScript/ggschema/src/data.ts

```diff
@@ -2,7 +2,7 @@
  * The datasets should have at least one dataset
  * @minProperties 1
  */
-export interface Datasets {
+export interface DatasetsObject {
```

Comment:
Could this be renamed to `DatasetObject`? This would follow on to the variable name `datasetObject` and also follow to ggvega

Reply:
Hi @ijlyttle ,

I'm free this afternoon.

Reply:
Hi @ijlyttle ,

In ggvega, the variable is `datasetsObject`. And I change it to `datasetObject`. 

Also, rename `datasetsObject.ts` to `datasetObject.ts`.

Rename `Metadata` to `MetadataObject`. Re `Metadatum` to `Metadata`.

If I misunderstood, please remind me.

Reply:
Sounds good!